config/config.go

```diff
@@ -93,6 +113,8 @@ func (cfg Config) WithURL(rawURL string) Config {
 // New returns a Config initialized with given parameters. The returned Config
 // is not guaranteed to be safe: it must be validated using Config.Validate
 // before usage.
+//
+// TODO: remove this function: https://github.com/benchttp/runner/pull/40/
```

Comment:
What's up with that ? Wasn't `config.New` not used anymore ?

Reply:
Well... As we can see in #40, I forgot to actually remove it 😬